src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_StorageFile2.cs

```diff
@@ -0,0 +1,89 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+
+// Given_StorageFile2 , as Given_StorageFile is used in PR#2407
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+    [TestClass]
+    public class Given_StorageFile2
+    {
+        
+        String _filename;
+        
+        [TestInitialize]
+        public void Init()
+        {
+            _filename = DateTime.Now.ToString("yyyyMMddHHmmss") + ".txt";
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+        }
+
+        [TestMethod]
+        public async void When_DateCreated()
+        {
+            
+            var _folder = Windows.Storage.ApplicationData.Current.LocalFolder;
+            Assert.IsNotNull(_folder, "cannot get LocalFolder - error outside tested method");
+
+            Windows.Storage.StorageFile _file = null;
+            
+            DateTimeOffset _dateBefore = DateTimeOffset.Now;
```

Comment:
Naming is pretty confusing in the method. Don't hesitate to use longer names to clarify their usage...  By example, it could be `dateBeforeCreatingFile`.

Also please reserve the `_` prefix for members. Local variables should use camel casing.